### DISCUSSION TITLE
Using zed ai in local debug builds

### DISCUSSION THREAD

**Discussion by KorigamiK** [zed-industries/zed#18883] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

Is it possible to use the Claude Sonnet model from zed ai in local builds?
Is there an api for zed ai that could be used externally?

### Environment

Linux 6.11.2-arch1-1

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
